templates/cloudfront.cfn.yml.j2

```diff
@@ -61,28 +61,33 @@ Resources:
     Properties:
       CachePolicyConfig:
         Comment: !Sub "${Service} Cache policy"
-        DefaultTTL: 1
-        MaxTTL: 1
-        MinTTL: 0
+        DefaultTTL: 5
+        MaxTTL: 31536000
+        MinTTL: 2
         Name: !Sub "${AWS::StackName}_CachePolicyCMS"
         ParametersInCacheKeyAndForwardedToOrigin:
           CookiesConfig:
-            CookieBehavior: "all" #none, whitelist, allExcept, all
+            CookieBehavior: "allExcept" #none, whitelist, allExcept, all
+            Cookies:
+              - "_dc_gtm_UA-7276966-13" #GTM/Google Analytics
+              - "_ga" #GTM/Google Analytics
+              - "_ga_6KF13E15B4" #GTM/Google Analytics
+              - "_gid" #GTM/Google Analytics
+              - "monsido" #Monsido
+              - "nmstat"
           EnableAcceptEncodingBrotli: true
           EnableAcceptEncodingGzip: true
           HeadersConfig:
             HeaderBehavior: "whitelist" #none, whitelist
             Headers: #max 10
```

Comment:
with 
``OriginRequestPolicyId: "33f36d7e-f396-46d9-90e0-52428a34d9dc" #AllViewerAndCloudFrontHeaders-2022-06``
we can now cut this down to how we want to cache and let the server get all when its a miss.